### PR TITLE
Feature/sentry support

### PR DESCRIPTION
**Pull request by burhan**
Adding sentry support for #194 

Note that I have left the URL as an argument, in case the Sentry server is local and not the hosted version at getsentry.com

All other settings have also been integrated, including the middleware.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/config/settings/production.py [hunk 1]
```diff
@@ -5,11 +5,13 @@
 - Use djangosecure
 - Use Amazon's S3 for storing static files and uploaded media
 - Use mailgun to send emails
-- Use MEMCACHIER on Heroku
+- Use Redis on Heroku
 '''
+{% if cookiecutter.use_sentry == "y" %}
+import raven
+{% endif %}
```
Comment (luzfcb) [inline {{cookiecutter.repo_name}}/config/settings/production.py]: __future__ import must be the first import the module

**Comment by luzfcb** [comment]
please, update the tests https://github.com/pydanny/cookiecutter-django/blob/master/tests/base.py#L18-L28 and README.rst usage section

**Comment by burhan** [comment]
Hi @luzfcb - README.rst was updated and tests also pass; I am working on a different PR to update the tests for the remaining features. There is some refactoring that can be done.

File: {{cookiecutter.repo_name}}/config/wsgi.py [hunk 2]
```diff
@@ -36,6 +39,9 @@
 # See: https://whitenoise.readthedocs.org/
 application = DjangoWhiteNoise(application)
 {%- endif %}
+{% if cookiecutter.use_sentry == "y" -%}
+application = Sentry(application)
```
Comment (luzfcb) [inline {{cookiecutter.repo_name}}/config/wsgi.py]: is this really necessary? if use_whitenoise is enable, this overwrites the previous `application`
some documentation about it?

**Comment by luzfcb** [comment]
@pydanny @theskumar ok for me.

**Pull request merged**